lib/extensions/test_swatches.py

```diff
@@ -5,51 +5,103 @@
 
 from inkex import errormsg
 
+from ..elements import FillStitch, SatinColumn, Stroke
+from ..gui.test_swatches import GenerateSwatchesApp
 from ..i18n import _
-from ..svg.tags import EMBROIDERABLE_TAGS
+from ..utils.param import ParamOption
 from .base import InkstitchExtension
 
 
 class TestSwatches(InkstitchExtension):
     '''
-    This generates swatches from selection by altering one param each time.
+    This saves embroider settings into the metadata of the file
```

Comment:
Is this correct?

Reply:
Hups, how did that happen. Thanks for pointing this out.